params.json

```diff
@@ -0,0 +1,8 @@
+{
+  "ParliamentWF.threads": "4",
```

Comment:
Is there a reason for `threads` her set to 4?
`parliament2.wdl` is pointing to a runtime instance with 32 cores. Can we set the default here to 32?
At line 96 of `parliament2.sh` the script is reserving 3 threads. If we set default to 4, would this affect to be only one thread being used for running parliament2?

Reply:
This is just an example script that is used for testing locally on my laptop; the `runtime:cpu` argument is ignored by WDL locally. I agree this is a bad value though - we should probably set it dynamically in the WDL.

I'm not sure how the thread reservation on line 96 (now 90) works exactly - does the core parliament2 script need three threads, or can we reclaim some of those?